CHANGES.rst

```diff
@@ -76,6 +76,8 @@ Cubeviz
 Imviz
 ^^^^^
 
+- Remove "From File.." option when running on an external server. [#3239]
```

Comment:
Actually do we want to advertise this?

Reply:
it'll be in the PR history regardless 🤷‍♂️

Reply:
If you insist on change log, I think also need entry under 4.0.x section.

Reply:
there is (this one you're commenting on is 4.0.1, the other one is 3.10.4)